health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/AdvancedMapCharts/HeatMap/HeatMapChart.js

```
@@ -0,0 +1,391 @@
+        }}
+      >
+        {range("0")}
+        <div
+          style={{
+            height: "14px",
+            background: "linear-gradient(270deg, #01D66F 0%, #FFC42E 46.03%, #FF7373 101.5%)",
+            width: "90%",
+            marginLeft: "10px",
+            marginRight: "10px",
+          }}
+        ></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)